tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py34,py35,packaging,pep8
+envlist = py27,py34,py35,py36,packaging,pep8
```

Comment:
Could you leave this off for now, we might look at it in the future for a PR but right now I don't want to start supporting 3.6 without a proper review.